Sources/TuistAutomation/Device/DeviceController.swift

```diff
@@ -8,17 +8,20 @@ import XcodeGraph
 
 enum DeviceControllerError: FatalError {
     case applicationVerificationFailed
+    case listDecodeFailed
```

Comment:
```suggestion
    case failedFetchingDevices
```

Let's align the error name and its description